LoopKit/Service/VersionCheckService.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  VersionCheckService.swift
+//  LoopKit
+//
+//  Created by Rick Pasetto on 9/8/21.
+//  Copyright © 2021 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+// Note: order is important for VersionUpdate.  Later version updates are more critical than earlier ones.  Do not reorder!
+public enum VersionUpdate: Comparable, CaseIterable {
+    /// No version update needed (i.e. running the latest version).
+    case noneNeeded
+    /// The version is unsupported; the app needs to be updated to the latest "supported" version.  Not a critical update.
+    case supportedNeeded
+    /// The version is bad and may be risky; the app must be updated immediately to avoid harm.
```

Comment:
[nit] This comment may imply certain things to the FDA. I think I'd just use something generic without indicating any cause like "The app must be updated immediately."